apps/angular/crud/src/app/store/tasks/task.effects.ts

```diff
@@ -63,9 +63,9 @@ export class TasksEffects {
   delTaskEffect = createEffect(() =>
     this.actions$.pipe(
       ofType(taskActions.delTaskAction),
-      switchMap((action) => {
+      exhaustMap((action) => {
         return this.service.delEntity(action.task).pipe(
-          switchMap((data) => {
+          mergeMap((data) => {
```

Comment:
why do you do a mergeMap with a of?

simply do map

Reply:
I do mergeMap because i want to fatten to different actions as a response. Update list and show a confirmation message. Maybe this is not the best approach, I'll be glad to know another option